### PR TITLE
first try with jest

### PR DESCRIPTION
**Pull request by aspirew**

### CODE REVIEW

File: homework.test.js [hunk 1]
```diff
@@ -28,5 +28,71 @@ async function homework(a, b, send, next) {
 }
 
 describe('Homework unit tests', () => {
-    //TODO: Implement unit tests that covers every edge case
-})
\ No newline at end of file
+        const a = 5
+        const b = 10
+        const send = jest.fn(x => x)
+        const next = jest.fn(x => x)
```
Comment (dimitor115) [inline homework.test.js:37]: Pierwsza ważna sprawa jest taka, że wartości początkowe dla testu powinniśmy deklarować per test. Każdy test jest oddzielną całością i musi mieć swoje wartości początkowe. Choćby dlatego, że jeśli współdzielisz pomiędzy testami mocki funkcji `send` i `next` to w każdym kolejnym teście będą się zmieniała liczba ich wywołań itd. To może być pierwszy powód dla którego coś Ci nie działa

File: homework.test.js [hunk 2]
```diff
@@ -28,5 +28,71 @@ async function homework(a, b, send, next) {
...
+
+    test('Should call send once with b as parameter when axios.get returns number bigger than b or smaller than a', () => {
+        jest.spyOn(axios, 'get').mockResolvedValue(b+4)
+        expect(send.mock.calls.length).toBe(1)
```
Comment (dimitor115) [inline homework.test.js:42]: Brakuje wywołania funkcji którą testujesz ;) Jak możesz oczekiwać jakiś wyników jej pracy jak jej nawet nie wywołałeś

File: homework.test.js [hunk 3]
```diff
@@ -28,5 +28,71 @@ async function homework(a, b, send, next) {
...
+        expect(send.mock.calls[0][0]).toEqual(b)
+    })
+
+    test('Should call send once with b as parameter when axios.get returns number bigger than b or smaller than a', () => {
+        jest.spyOn(axios, 'get').mockResolvedValue(a-3)
+        expect(send.mock.calls.length).toBe(1)
```
Comment (dimitor115) [inline homework.test.js:55]: Tu też brakje

**Review by dimitor115 — CHANGES_REQUESTED** [review]
Przede wszystkim przenieś deklaracje każdego mocka i każdej wartości początkowej do ciała testu. Wtedy myślę, że większość zacznie działać. A i pamiętaj KAŻDY test musi mieć unikalną nazwę
